src/Components/HomePage/ProductElement.js

```diff
@@ -29,7 +28,7 @@ const ProductElement = ({id, data, showCategory, showShortDescription}) => {
             {showShortDescription ? <p>{short_description}</p>: null}
 
             <ProductHoverButton>
-                <Button>Add to cart</Button>
+                <Button propsForFunction={ data && {productId, data}}>Add to cart</Button>
                 <Link to={`/product/${productId}`}>
                     <Button>View details</Button>
```

Comment:
Ideally the buttons should have an action, this should be either a paragraph, or a `div` component.